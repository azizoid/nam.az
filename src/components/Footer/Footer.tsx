import { version } from '../../../package.json'

export const Footer = () => (
  <nav className="w-full bg-gray-700 py-3 text-slate-300">
    <ol className="container mx-4 flex px-0 md:mx-auto">
      <li className="mr-6" aria-current="page">
        &copy; {new Date().getFullYear()}
      </li>
      <li className="mr-6">
        <a href="https://www.nam.az">Nam.az</a>
      </li>
      <li className="mr-6">
        <a href="https://www.quran.az" target="_blank">
          Quran.az
        </a>
      </li>
      <li className="mr-6">
        <a href="/credits" target="_blank">
          Təşəkkürlər
        </a>
      </li>
      <li className="ml-auto">v{version}</li> {/* Pushes the version to the right */}
    </ol>
  </nav>
)