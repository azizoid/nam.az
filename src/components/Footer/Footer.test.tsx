import { render } from '@testing-library/react'

import { Footer } from './Footer'

test('renders Footer component', () => {
  const { container } = render(<Footer />)

  expect(container).toMatchSnapshot()
})
