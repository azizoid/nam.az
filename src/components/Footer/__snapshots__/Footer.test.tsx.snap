// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer component 1`] = `
<div>
  <nav
    class="w-full bg-gray-700 py-3 text-slate-300"
  >
    <ol
      class="container mx-4 flex px-0 md:mx-auto"
    >
      <li
        aria-current="page"
        class="mr-6"
      >
        © 
        2023
      </li>
      <li
        class="mr-6"
      >
        <a
          href="https://www.nam.az"
        >
          Nam.az
        </a>
      </li>
      <li
        class="mr-6"
      >
        <a
          href="https://www.quran.az"
          target="_blank"
        >
          Quran.az
        </a>
      </li>
      <li
        class="mr-6"
      >
        <a
          href="/credits"
          target="_blank"
        >
          Təşəkkürlər
        </a>
      </li>
      <li
        class="ml-auto"
      >
        v
        1.0.1
      </li>
       
    </ol>
  </nav>
</div>
`;
